### DISCUSSION TITLE
DISCUSSION: DataFusion Meetup in India

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#10351] [Ideas]
After our first meetup, (see https://github.com/apache/datafusion/discussions/8152 / https://github.com/apache/datafusion/discussions/8522) I have heard there may be interest in other meetups in different geographies. 

I would personally love to support such meetups (will promote, try to attend, can help find sponsors, etc) as much as possible but I don't have the bandwidth to actually organize them.

Anyone who is interested in attending or helping to organize please leave a comment

Other meetup discussions:
https://github.com/apache/datafusion/discussions/10347

**Comment by vaibhawvipul** [comment]
I would love to help in organising.

**Reply by JyotinderSingh** [reply]
Would love to attend if it happens in india

**Reply by sivanagaraju** [reply]
Like to join the meetup

**Reply by neeleshbisht99** [reply]
I would also love to help in organizing.

**Reply by shiv4289** [reply]
Love to host or any help that would be required in organising this meetup.

**Reply by SmashYBoi04** [reply]
+1 I would also love to help in organizing.

**Reply by dharanad** [reply]
I am definitely interested in this and willing to help organise.

**Comment by neeleshbisht99** [comment]
I would also love to help in organizing.

**Comment by tusharchou** [comment]
@vaibhawvipul this would be great!

**Comment by vaibhawvipul** [comment]
@shiv4289 @nitisht Shall we plan about organising this in late June/early July?

We can look for speakers once we have a tentative date.

**Reply by nitisht** [reply]
Early July would be ideal @vaibhawvipul
